### PR TITLE
Add lang file for Firefox 67 whatsnew

### PR DESCRIPTION
**Pull request by craigcook**
See bedrock template at https://github.com/mozilla/bedrock/pull/7051

@peiying2 I've made an attempt but fully expect I got something wrong or missed a step. Your patient guidance is much appreciated!

### CODE REVIEW

**Comment by craigcook** [comment]
So much for "final" copy... got some last-second changes that I've added in a second commit (we should squash when it merges). Still awaiting actual, real, true, FINAL approval but I hope this is it.

**Comment by craigcook** [comment]
> Still awaiting actual, real, true, FINAL approval but I hope this is it.

And it's approved. These strings are hereby frozen.

**Comment by peiying2** [comment]
@craigcook I have been in meetings non-stop today. I will get take a look this evening.

**Comment by craigcook** [comment]
Squashed to one commit.

**Comment by peiying2** [comment]
Looks good overall with a few things here and there. 
- product names: Monitor and Send when used alone. Maybe it's worth adding a comment about keeping in it English.
- log-in or login: I should hear back from Michelle H soon as we discussed this almost a month ago.

File: en-US/firefox/whatsnew_67.lang [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+;Read the articles you care about anytime, anywhere (even offline). And try Pocket’s Listen feature so you can absorb great content on the go.
+Read the articles you care about anytime, anywhere (even offline). And try Pocket’s Listen feature so you can absorb great content on the go.
+
+
+# CTA button
+;Get the Pocket App
+Get the Pocket App
+
+
+;Keep an eye on data breaches with Monitor
```
[the remark this answers is not in the document]
Comment (flodolo) [reply en-US/firefox/whatsnew_67.lang]: Just to confirm that Monitor should be kept in English, per product guidelines when we translated it.

File: en-US/firefox/whatsnew_67.lang [hunk 2]
```diff
@@ -0,0 +1,99 @@
+## NOTE: https://www-demo5.allizom.org/firefox/67.0/whatsnew/
+## URL: https://www-demo5.allizom.org/%LOCALE%/firefox/66.0/whatsnew/
```
[the remark this answers is not in the document]
Comment (craigcook) [reply en-US/firefox/whatsnew_67.lang]: oops, the dangers of copy and paste ;)

File: en-US/firefox/whatsnew_67.lang [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+;Keep an eye on data breaches with Monitor
+Keep an eye on data breaches with Monitor
+
+
+;Find out if your information has been compromised by a data breach – and now you can sign up for alerts about future breaches with a Firefox account.
+Find out if your information has been compromised by a data breach – and now you can sign up for alerts about future breaches with a Firefox account.
+
+
+# CTA link points to https://monitor.firefox.com
+;Check Out Monitor
```
[the remark this answers is not in the document]
Comment (craigcook) [reply en-US/firefox/whatsnew_67.lang]: Monitor should be English as a proper name. I'll add a comment. 

And I've been told CTAs are title case so "Out" should be capitalized.

File: en-US/firefox/whatsnew_67.lang [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+## URL: https://www-demo5.allizom.org/%LOCALE%/firefox/66.0/whatsnew/
+
+
+# HTML page title
+;Get a Firefox Account – Keep your data private, safe and synced
+Get a Firefox Account – Keep your data private, safe and synced
+
+
+# HTML page description
+;Securely sync your passwords, bookmarks and tabs across all your devices. Get a Firefox Account now – One log-in – Power and privacy everywhere.
```
[the remark this answers is not in the document]
Comment (peiying2) [reply en-US/firefox/whatsnew_67.lang]: Michelle H confirmed that it should be `login` and that she has gotten confirmation from marketing team that they would follow the guidelines.

File: en-US/firefox/whatsnew_67.lang (hunk 1 above)
[the remark this answers is not in the document]
Comment (craigcook) [reply en-US/firefox/whatsnew_67.lang]: Added comment.

File: en-US/firefox/whatsnew_67.lang (hunk 3 above)
[the remark this answers is not in the document]
Comment (craigcook) [reply en-US/firefox/whatsnew_67.lang]: Added comment

File: en-US/firefox/whatsnew_67.lang [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+;Find out if your information has been compromised by a data breach – and now you can sign up for alerts about future breaches with a Firefox account.
+Find out if your information has been compromised by a data breach – and now you can sign up for alerts about future breaches with a Firefox account.
+
+
+# CTA link points to https://monitor.firefox.com
+;Check Out Monitor
+Check Out Monitor
+
+
+;Share large files securely with Send
```
[the remark this answers is not in the document]
Comment (craigcook) [reply en-US/firefox/whatsnew_67.lang]: Added comment

File: en-US/firefox/whatsnew_67.lang (hunk 4 above)
[the remark this answers is not in the document]
Comment (craigcook) [reply en-US/firefox/whatsnew_67.lang]: Updated to "login"

**Comment by peiying2** [comment]
@craigcook looks good. Now we need to add this new file to here and include name of the file, supported locale, deadline, priority etc. : https://github.com/mozilla-l10n/langchecker/blob/master/app/config/sources.inc.php

**Comment by peiying2** [comment]
@craigcook on Firefox account or Firefox Account, we should be consistent.

**Comment by craigcook** [comment]
> @craigcook on Firefox account or Firefox Account, we should be consistent.

I've been told "account" should be lowercase, with the reasoning that a "Firefox account" is not being treated as a product of its own, it's simply how someone accesses our other products/services. This seems like a change to the convention so although we have capitalized it in the past I don't think we will going forward.

I'm also told an official styleguide is in the works but I don't know the current status or when we might expect it to be published.

**Comment by peiying2** [comment]
> I've been told "account" should be lowercase, with the reasoning that a "Firefox account" is not being treated as a product of its own, it's simply how someone accesses our other products/services. This seems like a change to the convention so although we have capitalized it in the past I don't think we will going forward.

Checked with Michelle H again, she said to use Firefox Account (upper case). She got reassurance that marketing team would use the style guide!

**Comment by craigcook** [comment]
> Checked with Michelle H again, she said to use Firefox Account (upper case). She got reassurance that marketing team would use the style guide!

After a few rounds of discussion with a few different people, the consensus landed back on capitalizing "Firefox Account" since that's the current standard in Photon's style guide. That might change in the future as the marketing strategy evolves but for now I've changed it back to capital As.

**Comment by craigcook** [comment]
Updated with all locales thanks to @alexgibson and squashed

**Comment by flodolo** [comment]
Note that there's no actual need to squash, we normally squash when merging.

**Comment by peiying2** [comment]
@flod, for @craigcook to make production push, he needs to have access to the repo.

**Comment by flodolo** [comment]
> @flod, for @craigcook to make production push, he needs to have access to the repo.

Invite sent

**Pull request merged**